src/main/java/core/basesyntax/fruitstorage/FruitStorage.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.fruitstorage;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.fruitmodel.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitStorage implements FruitDao {
```

Comment:
Dao and Storage should be separated since storage represents db

Reply:
done